exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,51 @@
+#include "example.h"
+#include <vector>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
+    const std::vector<std::string> ordinals = {
+        "", "first", "second", "third", "fourth", "fifth", "sixth",
+        "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"
+    };
+    
+    const std::vector<std::string> gifts = {
+        "", "a Partridge in a Pear Tree", "two Turtle Doves", "three French Hens",
+        "four Calling Birds", "five Gold Rings", "six Geese-a-Laying",
+        "seven Swans-a-Swimming", "eight Maids-a-Milking", "nine Ladies Dancing",
+        "ten Lords-a-Leaping", "eleven Pipers Piping", "twelve Drummers Drumming"
+    };
+    
+    std::string result = "On the " + ordinals[day] + " day of Christmas my true love gave to me: ";
+    
+    for (int i = day; i >= 1; --i) {
+        if (i == day) {
+            result += gifts[i];
+        } else if (i == 1) {
+            result += ", and " + gifts[i];
```

Comment:
Same here:
```suggestion
            result += std::string{", and "} + gifts[i];
```
Alternatively, you can split it in two lines (this works because now we use `operator+=` of `result` (of type `std::string`), which has an overload that accepts `const char*`:
```suggestion
            result += ", and ";
            result += gifts[i];
```